### PR TITLE
Enhancement - Manager access to time-sheet management and leave approval 

### PR DESCRIPTION
**Pull request by vaibhav8186**

### CODE REVIEW

File: app/controllers/leave_applications_controller.rb [hunk 1]
```diff
@@ -95,7 +96,7 @@ def process_leave
   private
 
   def authorization_for_admin
-    if !current_user.role?("Admin")
+    if !(current_user.is_Admin || current_user.is_Manager)
```
Comment (swapnilchincholkar) [inline app/controllers/leave_applications_controller.rb]: @vaibhav8186 
Can we have method names like "is_admin?", "is_manager?"

File: app/models/user.rb [hunk 2]
```diff
@@ -112,7 +112,13 @@ def is_employee_or_intern?
 
   def is_admin_or_hr?
     [ROLE[:HR], ROLE[:admin]].include?(role)
-  end  
+  end
+
+  ["Admin", "Manager"].each do | method |
+    define_method "is_#{method}" do
```
Comment (swapnilchincholkar) [inline app/models/user.rb]: @vaibhav8186 Convert methods in way that they should be like "is_admin?", "is_manager?"

File: app/views/leave_applications/view_leave_status.html.haml [hunk 3]
```diff
@@ -5,7 +5,8 @@
   .leave_link{float: right;}
 .alert-message
 .leave_link
-  =link_to "Apply Leave", new_user_leave_application_path(current_user)
+  =link_to "Apply Leave", new_user_leave_application_path(current_user), class: 'btn btn-primary'
+  %h5 Available Leaves:#{@available_leaves}
```
Comment (swapnilchincholkar) [inline app/views/leave_applications/view_leave_status.html.haml]: @vaibhav8186 Need a space between : & count of available leaves

File: app/controllers/leave_applications_controller.rb (hunk 1 above)
Comment (vaibhav8186) [reply app/controllers/leave_applications_controller.rb]: yes. changed method names

**Pull request merged**